spec/unit/shoulda/matchers/active_model/validate_inclusion_of_matcher_spec.rb

```diff
@@ -212,6 +212,14 @@ def add_outside_value_to(values)
         end
       end
 
+      it 'matches when validation uses given message with %{value}' do
```

Comment:
I guess what I mean is, maybe we shouldn't be so specific here in the name of the test. It's not so much that `%{value}` is being used in the message. There's a standard list of variables you can use in your custom message that when ActiveRecord populates the errors hash for your record it will replace (`value`, `record`, `attribute` ... I'm probably missing one more). So it sounds like the bug is, if you use any one of these, they don't get replaced correctly. So I'm suggesting that we name the test to accurately describe the bug, that's all.